docs/tutorials/uniq-factories/factory-management/uniq-swap-example-project.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Uniq Swap Example Project'
+
+order: 4
+---
+
+
+# Uniq Swap Example Project
+
+This page covers an example project that lets users swap their Uniqs using first hand purchase functionality. For installation instructions refer to the repository itself.
+
+Code is available here: https://github.com/ultraio/example-first-hand-purchase
+
+## Use cases
+
+- This project provides a reference implementation for integrating first hand purchase functionality into your web-based solution.
+
+- Parts of the implementation can be extracted and reused as needed or you can simply update the code to your needs and deploy it as is.
+
+- If your project requires users to be able to get Uniqs in any way without having a backend then this example project can benefit you as it operates as a static website and interacts with Ultra Blockchain directly.
+
```

Comment:
One of the reasons we had started to work on this was "transferability/trade-ability" and new possibilities and being able to exchange old uniq with new one. We need to mention this in use cases as well.  https://ultraio.atlassian.net/browse/BLOCK-1950